tests/testthat/test-matrix.r

```diff
@@ -169,3 +177,29 @@ test_that("warning is thrown if it is assumed that the survey is representative"
   expect_warning(contact_matrix(survey=polymod4, symmetric=TRUE), "Assuming the survey is representative")
 })
 
+
+test_that("Taking mean of estimated contact's age give na when mean is not in an age limit ",
+{
+  cm <- contact_matrix(survey=polymod9, age.limits=c(0, 5, 10, 15, 20))
+  expect_true(is.na(rowSums(cm$matrix)[1]))
+  expect_false(is.na(rowSums(cm$matrix)[2]))
+  expect_true(is.na(rowSums(cm$matrix)[3]))
+  expect_true(is.na(rowSums(cm$matrix)[4]))
+  expect_false(is.na(rowSums(cm$matrix)[5]))
+  
+})
+test_that("Taking sample of estimated participant's give na when no overlap with the age limits ",
+{
+  cm <- contact_matrix(survey=polymod9, symmetric=TRUE, age.limits=c(0, 5, 10, 15, 20), estimated.participant.age = "sample")
```

Comment:
This triggers a warning. Can we remove `symmetric=TRUE`?

Reply:
Removed